iceberg/openhouse/internalcatalog/src/main/java/com/linkedin/openhouse/internal/catalog/fileio/FileIOConfig.java

```diff
@@ -91,4 +94,26 @@ S3FileIO provideS3FileIO() {
           "S3 storage configured but could not initialize S3FileIO" + e.getMessage());
     }
   }
+
+  /**
+   * Provides the ADLSFileIO bean for ADLS storage type
+   *
+   * @return ADLSFileIO bean for ADLS storage type, or null if ADLS storage type is not configured
+   */
+  @Bean
+  ADLSFileIO provideADLSFileIO() {
+    try {
+      // get the already-created FileIO from the ADLS Manager
+      ADLSFileIO fileIO = new ADLSFileIO();
+      Map props =
+          ((AdlsStorageClient) storageManager.getStorage(StorageType.ADLS).getClient())
```

Comment:
lets remove type casting, we should be able to do ` storageManager.getStorage(StorageType.ADLS).getProperties()` instead

Reply:
Done